doc/source/ray-contribute/development.rst

```diff
@@ -248,11 +332,43 @@ update the documentation!
 When you make documentation changes, build them locally to verify they render
 correctly. `Sphinx <http://sphinx-doc.org/>`_ is used to generate the documentation.
 
+Building Docs for Apple Silicon (M1)
```

Comment:
Quick question: Is it possible to entirely move / consolidate the "Building the Docs" section into `ray-contribute/docs.rst`? I think this might just be a little redundant (not necessarily a bad thing, but this document is already super long!)

Reply:
I hadn't seen that, I see that one is a Jupyter Notebook. Let me try to move those parts there.

Reply:
I think I got it right. 🤓  I moved all the docs about building the docs to that section/file and made a note on the main development page pointing to that one.

Reply:
awesome, thank you!